yoti-sdk-api/src/main/java/com/yoti/api/client/ActivityDetails.java

```diff
@@ -39,6 +39,14 @@
      */
     String getRememberMeId();
 
+    /**
+     * Return the parent-remember-me ID, which is a unique, stable identifier for a user in the context of an organisation/user
```

Comment:
Remember Me ID is `rememberMeId` in the developer docs, so do you think we should format it as either `ParentRememberMeID` or `Parent-Remember-Me-ID` here? Looks a bit disjointed currently

Reply:
Ok, renamed to rememberMeId & parentRememberMeId